ci-operator/config/devfile/integration-tests/devfile-integration-tests-master__v4.5.yaml

```diff
@@ -30,7 +30,27 @@ tests:
     cluster_profile: aws
     test:
     - as: integration-e2e-steps
-      commands: scripts/openshiftci-presubmit-all-tests.sh
+      commands: scripts/openshiftci-presubmit-devfiles-tests.sh
+      credentials:
+      - mount_path: /usr/local/ci-secrets/odo-rabbitmq
+        name: odo-rabbitmq
+        namespace: test-credentials
+      env:
+      - default: /usr/local/ci-secrets/odo-rabbitmq/amqpuri
+        name: ODO_RABBITMQ_AMQP_URL
```

Comment:
@josephca These credentials are not require from devfile repo point view. BTW you are not mapping any credentials for devfile integration repo.

Please remove these credentials and env fields.